app/src/test/java/protect/card_locker/LoyaltyCardViewActivityTest.java

```diff
@@ -488,12 +488,13 @@ public void checkMenu() throws IOException
         final Menu menu = shadowOf(activity).getOptionsMenu();
         assertTrue(menu != null);
 
-        // The share, settings and add button should be present
-        assertEquals(menu.size(), 3);
+        // The share, settings, add and star button should be present
+        assertEquals(menu.size(), 4);
 
         assertEquals("Block Rotation", menu.findItem(R.id.action_lock_unlock).getTitle().toString());
         assertEquals("Share", menu.findItem(R.id.action_share).getTitle().toString());
         assertEquals("Edit", menu.findItem(R.id.action_edit).getTitle().toString());
+        assertEquals("Remove from favorites", menu.findItem(R.id.action_star_unstar).getTitle().toString());
```

Comment:
But starred is 0 in this loyalty card. Shouldn't this be "Add to favorites" then?

Reply:
You are right! This is alraedy wrong in LoyaltyCardViewActivity.java